integration-tests/gradle/build.gradle

```diff
@@ -34,6 +34,7 @@ dependencies {
     testImplementation 'org.assertj:assertj-core:3.16.1'
     testImplementation 'org.junit.jupiter:junit-jupiter:5.6.2'
     testImplementation 'org.awaitility:awaitility:4.0.3'
+    testImplementation "org.projectlombok:lombok:${lombokVersion}"
```

Comment:
Is this dependency needed here?

Reply:
The only reason I added it here was to pre-download it. But it's not needed. I'll probably remove it. Figuring out why the CI fails. Perhaps another maven local repo path related issue.